x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/SecurityTests.java

```diff
@@ -146,8 +146,6 @@ protected SSLService getSslService() {
     @Before
     public void cleanup() throws IOException {
         if (threadContext != null) {
-            threadContext.stashContext();
-            threadContext.close();
             threadContext = null;
         }
```

Comment:
we could remove the `if` statement?

Reply:
Good catch. I removed it and made it an after method, which makes more sense to me than a Before method.